dummyisolator/isolator.cpp

```diff
@@ -14,15 +14,17 @@
 #include <stout/hashmap.hpp>
 #include <stout/option.hpp>
 
-using namespace mesos;
+#include <list>
+#include <string>
+
+// TODO(nnielsen): Break into explicit using-declarations instead.
+using namespace mesos;  // NOLINT(build/namespaces)
 
 using mesos::slave::Isolator;
 
-class DummyIsolatorProcess : public mesos::slave::IsolatorProcess
-{
-public:
-  static Try<mesos::slave::Isolator*> create(const Parameters& parameters)
-  {
+class DummyIsolatorProcess : public mesos::slave::IsolatorProcess {
+ public:
+  static Try<mesos::slave::Isolator*> create(const Parameters& parameters) {
```

Comment:
One question.. To fix linter issues you put braces in the end of line in functions instead of next line. That breaks mesos style guide and goes with strict google style. Is this on purpose, or is it the linter not willing to be configured other way? (: Just being curious.